km_api/know_me/models.py

```diff
@@ -355,6 +393,111 @@ def has_object_write_permission(self, request):
         return request.user == self.user
 
 
+class EmergencyContact(mixins.IsAuthenticatedMixin, models.Model):
+    """
+    An emergency contact holds contact information for emercency situations
+
+    Attributes:
+        km_user:
+            Emergency Contacts parent ``KMUser``.
+        name (str):
+            The emergency contacts's name.
+        relation (str):
+            Emergency contats relationship with the profile owner.
+        phone_number (big int):
+            The emergency contacts phone number.
+        alt_phone_number (optional):
+            Second optional phone number for the emergency contact.
+        email (optional):
+            The emergency contact's email address.
+    """
+    km_user = models.ForeignKey(
+        'know_me.KMUser',
+        related_name='emergency_contacts',
+        related_query_name='emergency_contact',
+        verbose_name=_('know me user'))
+    name = models.CharField(
+        max_length=255,
+        verbose_name=_('name'))
+    relation = models.TextField(
+        max_length=255,
+        verbose_name=_('relation'))
+    phone_number = models.CharField(
+        validators=[
+            RegexValidator(regex='^1?\d{9,15}$',
+                           message='Enter valid phone number',
+                           code='invaid_phone_number'), ],
+        max_length=15,
+        verbose_name=_('phone number'))
+    alt_phone_number = models.CharField(
+        validators=[
+            RegexValidator(regex='^1?\d{9,15}$',
+                           message='Enter valid phone number',
+                           code='invaid_phone_number'), ],
```

Comment:
invaid = invalid ? You may have meant to do invaid though and I am missing something.